Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Types/Person.hs

```diff
@@ -13,6 +13,7 @@
 -}
 {-# LANGUAGE ApplicativeDo #-}
 {-# LANGUAGE DerivingStrategies #-}
+{-# OPTIONS_GHC -Wno-type-defaults #-}
```

Comment:
Please remove this pragma. (You can to specify type explicitly like `round @Centesimal @Int`)